### PR TITLE
update docs for contributors

### PR DESCRIPTION
**Pull request by jflinchbaugh**
* add a pull request templatea
* mention lint in readme
* refine contributing.md to remove unnecessary steps

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,27 +18,25 @@ Looking to dive in? FANTASTIC! Reach out to one of our experts and we can help g
 * Hit us up on our Slack channel #integrations.
 * Please fill out an issue (Jira INT project) for your PR so that we have traceability as to what you are trying to fix,
   versus how you fixed it.
+* Add the `@sonatype/integrations` group as a reviewer on any PR.
 
 ### For Non-Sonatypers
 
-* Come hang out with us at our [gitter channel](https://gitter.im/sonatype/nexus-developers) so we can try to understand the problem you are trying to solve.
```
Comment (jflinchbaugh) [inline CONTRIBUTING.md:24]: No one appears to be watching gitter, so we're just wasting contributors' time.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#### Description of Change
+
+#### Things to Do Before Submitting
+
+* Have you signed the [Sonatype CLA](https://sonatypecla.herokuapp.com/sign-cla)?
+* Have you added and run automated tests for your change? (See the README.md.)
+* Have you run `helm lint` on your change?
```
Comment (jflinchbaugh) [inline .github/pull_request_template.md:7]: prompt good behaviors and put CLA link up front.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#### Description of Change
+
+#### Things to Do Before Submitting
+
+* Have you signed the [Sonatype CLA](https://sonatypecla.herokuapp.com/sign-cla)?
+* Have you added and run automated tests for your change? (See the README.md.)
```
Comment (joedragons) [inline .github/pull_request_template.md:6]: Trivial: Feel like README could be a link if you're feeling nice.  Also, don't know the grammar rules but the period inside the parenthesis looks weird.  If a distinct sentence, why even have the parenthesis?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -18,27 +18,25 @@ Looking to dive in? FANTASTIC! Reach out to one of our experts and we can help g
...
-* If you're stuck, ask our [gitter channel](https://gitter.im/sonatype/nexus-developers) or #integrations in Slack! There are a number of
-  experienced programmers who are happy to help with learning and troubleshooting.
 
-### PR commenting protocol
+### Protocol of Commenting on a Pull Request
 
 Here are the general rules to follow when commenting on PRs for this repo:
 
-* Unless stated otherwise by the commenter, or clearly not meant to be responded to, all comments on a PR are expected
+* Unless stated otherwise by the commenter, or clearly not meant to be responded to, all comments on a PR are expected.
```
Comment (joedragons) [inline CONTRIBUTING.md:38]: Errant?

File: .github/pull_request_template.md (hunk 3 above)
Comment (jflinchbaugh) [reply .github/pull_request_template.md]: done.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -18,27 +18,25 @@ Looking to dive in? FANTASTIC! Reach out to one of our experts and we can help g
...
   to be addressed before it is merged. Sometimes it is alright for a reviewer to approve the PR before all of their
-  comments are addressed, but generally only when those comments are expected to be easily addressable without further
+  comments are addressed, but generally only when those comments are expected to be easily addressable without further.
```
[the remark this answers is not in the document]
Comment (jflinchbaugh) [reply CONTRIBUTING.md]: yeah, thanks. the formatting confused me for a moment and i started dropping periods in the wrong places. fixed.

**Review by joedragons — APPROVED** [review]
Still means nothing but looks good :D

**Review by hectorhuol — APPROVED** [review]
+1

**Pull request merged**